photon-api/src/main/scala/com/linkedin/photon/ml/optimization/game/RandomEffectOptimizationProblem.scala

```diff
@@ -43,27 +43,57 @@ protected[ml] class RandomEffectOptimizationProblem[Objective <: SingleNodeObjec
     val isTrackingState: Boolean)
   extends RDDLike {
 
+  /**
+   * Get the Spark context.
+   *
+   * @return The Spark context
+   */
   def sparkContext: SparkContext = optimizationProblems.sparkContext
```

Comment:
Well, but in Scala, whether is a value or a function is intentionally made irrelevant... in other words, it makes no difference, so even "functions" should be `sc` IMHO. That one won't even be called with function call syntax, by the way.